src/vs/workbench/browser/parts/editor/media/multieditortabscontrol.css

```diff
@@ -50,7 +50,7 @@
 	display: none;
 }
 
-.monaco-workbench .part.editor > .content .editor-group-container > .title > .tabs-and-actions-container.tabs-border-bottom::after {
+.monaco-workbench .part.editor > .content .editor-group-container > .title .tab:not([tabIndex="0"]) .tabs-and-actions-container.tabs-border-bottom::after {
```

Comment:
@benibenj that is creative, but we have classes for the active tab, no?

Reply:
Agree, changing